.github/workflows/inspektor-gadget.yml

```diff
@@ -22,6 +22,7 @@ jobs:
 
     - name: Build binaries for the gadget container image
       run: |
+        cd gadget-docker
```

Comment:
It could be `make -C gadget-docker gadget-container-deps`.

I would name the directory `gadget-container` to make it more generic: it should hopefully be possible to make IG work with containerd/runc or other container runtimes.

Reply:
`gadget-container` is a much better name!. I updated the line regarding `make` and also added a new entry in the main Makefile to create the container image.

Reply:
LGTM!